patch-analyzer/pom.xml

```diff
@@ -24,9 +24,9 @@
 	<modelVersion>4.0.0</modelVersion>
```

Comment:
Why isn't the maven-resources-plugin part of this pom as well?

Reply:
The JAR will not be published/deployed anyways, and the entire patch-analyzer module will soon disappear

Reply:
Also added it here, even if the module will soon disappear